rfcs/019-platform_reliability/ci_cd.md

```diff
@@ -0,0 +1,47 @@
+# CI/CD
+
+Continuous Integration / Continuous deployment
+
+## Deployment
+
+###  Continuous deployment
+
+When test pass against a build in a CI environment those changes are deployed to production.
+
+We need:
+- To be satisfied that tests in CI ensure that a build is promotable to production.
+- To build on the release tool to ensure the deploy step can actually trigger deployments.
+- To connect tests to deployment!
```

Comment:
Can we add something here about rolling back, as if we have CD, the confidence in reverting quick is necessary.